radis/io/hitran.py

```diff
@@ -1379,6 +1377,9 @@ def download_all_hitran_isotopes(molecule, directory, extra_params):
             .. warning::
                 this won't be able to download higher isotopes (ex : isotope 10-11-12 for CO2)
                 Neglected for the moment, they're irrelevant for most calculations anyway
+            .. Isotope Missing:
+                When an isotope is missing for a particular molecule then a key error `(molecule_id, isotope_id)
+                is raised. So the regex are cryptic.
```

Comment:
This should be a comment in the code; where your use your regex to extract the integers
The sentence "so the regex are cryptic" is not required ;)

Reply:
Got it